examples/api/lib/widgets/scroll_view/list_view.1.dart

```diff
@@ -0,0 +1,98 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+/// Flutter code sample for a [ListView.custom] using the `findChildIndexCallback` argument
+
```

Comment:
nit: add a period at the end of the comment and remove the blank line so it's snug against `main`.

Style guide ref: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#use-correct-grammar